go.mod

```diff
@@ -4,7 +4,6 @@ go 1.20
 
 require (
 	github.com/AlekSi/pointer v1.2.0
-	github.com/SAP/go-hdb v1.1.1
```

Comment:
That one is needed

Reply:
This depends on which Go driver we would want to use. There is the one on Github and then there is the official HANA Go driver which can be found here https://tools.hana.ondemand.com/#hanatools. I have created now a script and figured how to set it up for Github Actions. 

I do not know the difference. So, I guess up to you whether it should be the Github or the official driver which we use.

Reply:
Well, currently, CI fails due to missing dependency

Reply:
Yes, I also did not add the script and the changes yet. More or less just found the solution 30 minutes ago :D 
But if we decide for the official driver, I can add it.

Reply:
Please see Slack